aim/cli/watcher_cli.py

```diff
@@ -0,0 +1,171 @@
+import click
+from click import core
+import uuid
+import json
+
+from aim.ext.notifier import get_config
+from aim.ext.notifier.utils import has_watcher_config, set_default_config
+from aim.sdk.run_status_watcher import RunStatusWatcher
+from aim.sdk.repo import Repo
+
+core._verify_python3_env = lambda: None
+DEFAULT_MESSAGE_TEMPLATE = "❗️ Something wrong with Run '{run_hash}'. Please check. ❗️"
+
+
+def check_configuration(repo: Repo):
+    if not has_watcher_config(repo.path):
+        if click.confirm(f'Repo {repo.path} has no configured notifications. Would you like to '
+                         f'use default configuration?', abort=True):
+            set_default_config(repo.path)
+
+
+@click.group(invoke_without_command=True)
+@click.option('--repo', required=False, type=click.Path(exists=True,
+                                                        file_okay=False,
+                                                        dir_okay=True,
+                                                        writable=True))
+@click.pass_context
+def cli_entry_point(ctx, repo):
```

Comment:
help messages would be quite helpful here. didn't get what "disable" and "dump" meant.
<img width="169" alt="image" src="https://user-images.githubusercontent.com/13848158/183982808-e54b7c4a-2c49-4cd7-82ec-dc4e172a6db8.png">

Reply:
also, i believe having `list` command would be helpful, as i don't know how to find the notifier id

Reply:
@gorarakelyan disable/enable are helpful when you want to temporarily switch off notifications for particular channel, without removing the setup